src/Features/CSharp/Portable/InternalUtilities/InternalExtensions.cs

```diff
@@ -23,7 +23,7 @@ internal static class InternalExtensions
             if (argument.Expression.Kind() == SyntaxKind.DeclarationExpression)
             {
                 var decl = (DeclarationExpressionSyntax)argument.Expression;
-                typeInfo = semanticModel.GetTypeInfo(decl.Type);
+                typeInfo = semanticModel.GetTypeInfo(decl.Type, cancellationToken);
```

Comment:
nice.  consider breaking out into another PR so this can go in regardless of the rest of hte PR.

Reply:
It's a separate commit, but yes will break it out if this doesn't go in (but I don't suspect it won't.)